### PR TITLE
[qt*] ios fixes

### PR DESCRIPTION
**Pull request by m-kuhn**
- No serialport on ios
 - No opengl on ios (it uses metal)

### CODE REVIEW

File: ports/qtbase/vcpkg.json [hunk 1]
```diff
@@ -62,7 +62,6 @@
     "icu",
     "jpeg",
     "network",
-    "opengl",
```
Comment (autoantwort) [inline ports/qtbase/vcpkg.json:65]: You can also use platform expressions here
```suggestion
{
  "name": "opengl",
  "platform": "!ios"
},
```

Comment (m-kuhn) [reply ports/qtbase/vcpkg.json:65]: And remove this part instead?

https://github.com/microsoft/vcpkg/pull/36003/files#diff-d285619e22ce186bd6d68cae7da0442cd759a9fd0e0234b253e7f06a4b4f19a9R148-R155

Comment (autoantwort) [reply ports/qtbase/vcpkg.json:65]: Yes

Comment (m-kuhn) [reply ports/qtbase/vcpkg.json:65]: Thanks

**Review by MonicaLiu0311 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks!

**Pull request merged**